MdeModulePkg/Core/Dxe/Image/Image.c

```diff
@@ -1189,22 +1221,30 @@ CoreLoadImageCommon (
   }
 
   if (EFI_ERROR (Status)) {
-    Image = NULL;
     goto Done;
   }
 
+  if (gBdsStarted) {
+    ImageOrigin = UefiImageOriginUserImage;
+  }
+
   //
   // Get information about the image being loaded
   //
   Status = UefiImageInitializeContextPreHash (
              &ImageContext,
              FHand.Source,
              (UINT32) FHand.SourceSize,
-             ImageIsFromFv
+             UEFI_IMAGE_SOURCE_FV,
+             ImageOrigin
              );
+  if (Status == EFI_NOT_STARTED) {

```

Comment:
Should this be EFI_UNSUPPORTED: https://github.com/acidanthera/audk/pull/58#issuecomment-1816474965

Reply:
It is distinguished deliberately according to policy https://github.com/acidanthera/audk/blob/uefi-format/MdePkg/Library/BasePeCoffLib2/PeCoffInit.c#L484-L503

Reply:
Yes, apologies, I misread UefiImageInitializeContextPreHash and thought it was only returning unsupported or success.